test/cloud-slack-dev-e2e/cloud_slack_dev_e2e_test.go

```diff
@@ -374,34 +374,46 @@ func TestCloudSlackE2E(t *testing.T) {
 			t.Log("Testing ping with --cluster-name")
 			command := fmt.Sprintf("ping --cluster-name %s", deployment.Name)
 			expectedMessage := fmt.Sprintf("`%s` on `%s`\n```\npong", command, deployment.Name)
-			tester.PostMessageToBot(t, channel.ID(), command)
+			tester.PostMessageToBot(t, channel.Identifier(), command)
 			err = tester.WaitForLastMessageContains(tester.BotUserID(), channel.ID(), expectedMessage)
 			require.NoError(t, err)
 
 			t.Log("Testing ping for not connected deployment #2")
 			command = "ping"
 			expectedBlockMessage := notConnectedMessage(deployment2.Name, deployment2.ID)
-			tester.PostMessageToBot(t, channel.ID(), fmt.Sprintf("%s --cluster-name %s", command, deployment2.Name))
+			tester.PostMessageToBot(t, channel.Identifier(), fmt.Sprintf("%s --cluster-name %s", command, deployment2.Name))
 
 			renderedMsg := interactive.RenderMessage(tester.MDFormatter(), expectedBlockMessage)
 			renderedMsg = strings.Replace(renderedMsg, "\n", " ", -1)
 			renderedMsg = strings.TrimSuffix(renderedMsg, " ")
 			err = tester.WaitForLastInteractiveMessagePostedEqualWithCustomRender(tester.BotUserID(), channel.ID(), renderedMsg)
+			if err != nil { // the new cloud backend not release yet
+				t.Logf("Fallback to the old behavior with message sent at the channel level...")
+				err = tester.OnChannel().WaitForLastInteractiveMessagePostedEqualWithCustomRender(tester.BotUserID(), channel.ID(), renderedMsg)
+			}
 			require.NoError(t, err)
 
 			t.Log("Testing ping for not existing deployment")
 			command = "ping"
 			deployName := "non-existing-deployment"
 			expectedMessage = fmt.Sprintf("*Instance not found* The cluster %q does not exist.", deployName)
-			tester.PostMessageToBot(t, channel.ID(), fmt.Sprintf("%s --cluster-name %s", command, deployName))
+			tester.PostMessageToBot(t, channel.Identifier(), fmt.Sprintf("%s --cluster-name %s", command, deployName))
 			err = tester.WaitForLastMessageContains(tester.BotUserID(), channel.ID(), expectedMessage)
+			if err != nil { // the new cloud backend not release yet
+				t.Logf("Fallback to the old behavior with message sent at the channel level...")
+				err = tester.OnChannel().WaitForLastMessageContains(tester.BotUserID(), channel.ID(), expectedMessage)
+			}
 			require.NoError(t, err)
 
 			t.Log("Setting cluster as default")
-			tester.PostMessageToBot(t, channel.ID(), fmt.Sprintf("cloud set default-instance %s", deployment.ID))
+			tester.PostMessageToBot(t, channel.Identifier(), fmt.Sprintf("cloud set default-instance %s", deployment.ID))
 			t.Log("Waiting for confirmation message...")
 			expectedClusterDefaultMsg := fmt.Sprintf(":white_check_mark: Instance %s was successfully selected as the default cluster for this channel.", deployment.Name)
 			err = tester.WaitForLastMessageEqual(tester.BotUserID(), channel.ID(), expectedClusterDefaultMsg)
+			if err != nil { // the new cloud backend not release yet
```

Comment:
We can remove the fallback later after release 🤔 Can you make a reminder for it? Thanks!